tools/downloader/converter.py

```diff
@@ -130,18 +164,28 @@ def main():
                 '--model_name={}'.format(top.name),
                 *expanded_mo_args]
 
-            print('========= {}Converting {} to IR ({})'.format(
-                '(DRY RUN) ' if args.dry_run else '', top.name, top_precision))
+            prefixed_printf(stdout_prefix, '========= {}Converting {} to IR ({})',
+                '(DRY RUN) ' if args.dry_run else '', top.name, top_precision)
 
-            print('Conversion command:', ' '.join(map(quote_arg, mo_cmd)))
+            prefixed_printf(stdout_prefix, 'Conversion command: {}', ' '.join(map(quote_arg, mo_cmd)))
 
             if not args.dry_run:
-                print(flush=True)
+                prefixed_printf(stdout_prefix, '', flush=True)
+
+                if not prefixed_subprocess(stdout_prefix, mo_cmd):
+                    return False
+
+            prefixed_printf(stdout_prefix, '')
 
-                if subprocess.run(mo_cmd).returncode != 0:
-                    failed_topologies.add(top.name)
+        return True
+
+    if args.jobs == 1:
```

Comment:
I suggest to run in one thread for dry-run also

Reply:
Excellent point; will do.

Reply:
Done.